### PR TITLE
fixed config manager deletion for 5.7

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
1. fixed delete flash message text for 5.7. it is changed there.
2. fixed typo Description->Details

### CODE REVIEW

File: cfme/infrastructure/config_management.py [hunk 1]
```diff
@@ -191,7 +191,11 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
 
         sel.handle_alert(cancel)
         if not cancel:
-            flash_msg = 'Delete initiated for 1 provider'
+            flash_msg = version.pick({'5.6': 'Delete initiated for 1 provider',
+                                      '5.7': 'Delete initiated for 1 Provider',
+                                      version.LATEST: 'Delete initiated for 1 Provider'
```
Comment (psav) [inline cfme/infrastructure/config_management.py]: 5.7 is same as version.LATEST, you don't need version.LATEST line

**Comment by dajoRH** [comment]
Lint report for commit e31c60f95929882d475512916261b210726d55d3:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/infrastructure/config_management.py [hunk 2]
```diff
@@ -185,13 +185,14 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
...
         item_text = version.pick({'5.6': 'Remove selected items from the VMDB',
-                                  '5.7': 'Remove selected items',
                                   version.LATEST: 'Remove selected items'})
         cfg_btn(item_text, invokes_alert=True)
 
         sel.handle_alert(cancel)
         if not cancel:
-            flash_msg = 'Delete initiated for 1 provider'
+            flash_msg = version.pick({'5.6': 'Delete initiated for 1 provider',
+                                      version.LATEST: 'Delete initiated for 1 Provider'})
```
Comment (psav) [inline cfme/infrastructure/config_management.py]: I think ```version.LATEST``` needs to be ```5.7```

Comment (izapolsk) [inline cfme/infrastructure/config_management.py]: I used LATEST in order to get rid of changing this again when we start testing 5.8+. are you still certain about changing this ?

**Comment by dajoRH** [comment]
Lint report for commit fca4bb6474df01fb70c3b7d00039ee6b9dfbff34:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**